public/css/poole.css

```diff
@@ -186,6 +186,10 @@ pre code {
 }
 .highlight pre {
   margin-bottom: 0;
+
+  /* add scrolling for code blocks which are too wide */
+  white-space: pre;
+  overflow: auto;
```

Comment:
Note that in `poole.css`, we find in lines 169 and following:
```
  white-space: pre;
  white-space: pre-wrap;
  word-break: break-all;
  word-wrap: break-word;
...
```
which makes no sense... perhaps remove the `white-space: pre-wrap;` there and the `word-wrap`, and then add in the `overflow: auto` there?